src/test/java/Homework18.java

```diff
@@ -0,0 +1,31 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework18 extends BaseTest {
+//    Create a test case, playSong() using @Test annotation.
+//    Use the helper/reusable methods we created.
+//    Create a new config XML file for the test case.
+//    Specify the method to be ran in the XML file.
+//    Navigate to "https://bbb.testpro.io/".
+//    Log in with your credentials.
+//    Click «Play next song» (media player controls), then the Play button, to play a song.
+//    Validate that a song is playing by verifying if the sound bar, or the pause button is displayed.
```

Comment:
**I recommend not using code comments:** for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.